test/cases/load_test.py

```diff
@@ -98,7 +98,7 @@ def _(environment, **kw):
         # Test separately the compose request and the other ones as they are not
         # expected to have the same latencies.
         if key[0] == '/api/image-builder/v1/compose':
-            if value.fail_ratio < COMPOSE_FAIL_RATIO:
+            if value.fail_ratio > COMPOSE_FAIL_RATIO:
                 print(f"{key} Test failed due to failure ratio "
                         f"< {COMPOSE_FAIL_RATIO}%")
```

Comment:
Could you fix this small typo?
```suggestion
                        f"> {COMPOSE_FAIL_RATIO}%")
```